### DISCUSSION TITLE
How do battery indicator and wifi icons decide in which panel they appear?

### DISCUSSION THREAD

**Discussion by PexoHundido69** [lxqt/lxqt#2186] [Q&A]
As you can see, I am using lxqt with i3. Battery and wifi icons moved to the i3 status bar and I don't know how to change it. I want them on the lxqt panel.
Is there any config file or something to change this? Here is a pic.

https://postimg.cc/56zKP2Xh

**Comment by stefonarch** [comment]
Afaik both go to status notifier area (ex systray). There can be only one.

**Reply by palinek** [reply]
Yes, if i3 also implements (registers) status notifier watcher it will win, as it will register first with dbus.
